src/content/docs/fr/guides/integrations-guide/cloudflare.mdx

```diff
@@ -0,0 +1,403 @@
+---
+type: integration
+title: '@astrojs/cloudflare'
+description:  Apprendre à utiliser l'adaptateur SSR @astrojs/cloudflare pour déployer votre projet Astro.
+githubURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/cloudflare/'
+hasREADME: true
+category: adapter
+i18nReady: true
+---
+
+import Video from '~/components/Video.astro';
+import DontEditWarning from '~/components/DontEditWarning.astro';
+
+<DontEditWarning/>
+
+Un adaptateur SSR pour utiliser les fonctions de Cloudflare Pages. Écrivez votre code en Astro/Javascript et déployez-le sur Cloudflare Pages.
+
+## Installation
+
+Ajoutez l'adaptateur Cloudflare pour activer le SSR dans votre projet Astro avec la commande `astro add`. Cela installera l'adaptateur et apportera les changements appropriés à votre fichier `astro.config.mjs` en une seule étape.
+
+```sh
+# En utilisant NPM
+npx astro add cloudflare
+# En utilisant Yarn
+yarn astro add cloudflare
+# En utilisant PNPM
+pnpm astro add cloudflare
+```
+
+Si vous préférez installer l'adaptateur manuellement, suivez les deux étapes suivantes :
+
+1.  Ajoutez l'adaptateur Cloudflare aux dépendances de votre projet à l'aide de votre gestionnaire de paquets préféré. Si vous utilisez npm ou si vous n'êtes pas sûr, exécutez ceci dans le terminal :
+
+```bash
+npm install @astrojs/cloudflare
+```
+
+2.  Ajoutez ce qui suit à votre fichier `astro.config.mjs` :
+
+```diff lang="ts"
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
++ import cloudflare from '@astrojs/cloudflare';
+
+  export default defineConfig({
++   output: 'server',
++   adapter: cloudflare(),
+  });
+```
+
+## Options
+
+### Mode
+
+`mode: "advanced" | "directory"`
+
+Par défaut `"advanced"`
+
+Cloudflare Pages a 2 modes différents pour déployer les fonctions, le mode `advanced` qui récupère le `_worker.js` dans `dist`, ou un mode répertoire où les pages vont compiler le worker à partir d'un dossier functions à la racine du projet. Pour la plupart des projets, l'adaptateur par défaut `advanced` sera suffisant ; le dossier `dist` contiendra votre projet compilé.
```

Comment:
This is purely optional:
For `un mode répertoire`, I think we should keep the mode name instead of its translation.
```suggestion
Cloudflare Pages a 2 modes différents pour déployer les fonctions, le mode `advanced` qui récupère le fichier `_worker.js` dans le dossier `dist`, ou un mode `directory` pour lequel les pages vont compiler le worker à partir d'un dossier functions à la racine du projet. Pour la plupart des projets, l'adaptateur par défaut `advanced` sera suffisant ; le dossier `dist` contiendra votre projet compilé.
```